core/src/components/card/sdds-card.stories.tsx

```diff
@@ -94,14 +94,14 @@ export default {
     },
     cardBottom: {
       name: 'Content of the bottom of the card',
```

Comment:
```suggestion
      name: 'Content of the bottom of the Card',
```
As @nathalielindqvist suggested ;)